lazyflow/operators/opReorderAxes.py

```diff
@@ -39,6 +39,13 @@ class OpReorderAxes(Operator):
     AxisOrder = InputSlot(value='tzyxc') # string: The desired output axis order
     Output = OutputSlot()
 
+    def __init__(self, graph=None, parent=None, Input=None, AxisOrder=None):
```

Comment:
So input should be a slot and axisorder a string then. What about connecting the output of a different operator that spits out an axiorder?

Reply:
True. I guess I should be accepting slots as well as strings here